packages/vaex-core/vaex/expression.py

```diff
@@ -538,12 +538,40 @@ def count(self, binby=[], limits=None, shape=default_shape, selection=False, del
         kwargs['expression'] = self.expression
         return self.ds.count(**kwargs)
 
-    def sum(self, binby=[], limits=None, shape=default_shape, selection=False, delay=False, progress=None):
+    def sum(self, axis=None, binby=[], limits=None, shape=default_shape, selection=False, delay=False, fill_missing=None, fill_missing_item=0, fill_empty=0, progress=None):
```

Comment:
@JovanVeljanoski I wonder if we should change the API and remove the binby/limits/shape arguments?

Reply:
I was kind of hoping that the "flagship" methods like count, sum, mean, std would keep the arguments they have for the time being. 

If we remove them, maybe we can "port" them to the `binby` method, kind of like the plotting, the include what/limits/shape etc.. (whatever makes sense).

But for now I would keep it.. has been a core feature for a long time..

Reply:
happy to keep them, was wondering if you had the same idea (df.sum will still have it btw)